Config/armbsp.cfg

```diff
@@ -0,0 +1,33 @@
+[gdbServer]
+address = %(ARMBSP_RASPI_ADDRESS)s
+userName = %(TASTE_RASPI_USER)s
+password = %(TASTE_RASPI_PASSWORD)s
+path= openocd
+args=   -f /usr/share/openocd/scripts/interface/cmsis-dap.cfg \
+        -f /usr/share/openocd/scripts/board/atmel_samv71_xplained_ultra.cfg \
+        -f /home/pi/broadcastOpenocd.cfg
+verbose = True
+
+[gdb]
+address = %(ARMBSP_RASPI_ADDRESS)s:%(TASTE_RASPI_PORT)s
+path = arm-none-eabi-gdb
+verbose = True
+
+[ioConsole]
+address = %(ARMBSP_RASPI_ADDRESS)s
+userName = %(TASTE_RASPI_USER)s
+password = %(TASTE_RASPI_PASSWORD)s
+baudrate = 115200
+port = 5005
+path = /dev/ttyACM0
+verbose = True
+
+[ioUart4]
+address = %(ARMBSP_RASPI_ADDRESS)s
+userName = %(TASTE_RASPI_USER)s
+password = %(TASTE_RASPI_PASSWORD)s
+baudrate = 38400
+port = 5006
+path = /dev/serial0
+vPortName = vUart4
```

Comment:
This applies not only here, but also across python code. Would it be possible to make better names? This is virtualPortName? virtualDeviceName? Maybe sync this renaming with path -> hardwarePortPath? hardwareDevicePath? What do you think?

Reply:
I agree. I will try to fix it across the whole project.